### ISSUE TITLE
hts2png not working

### ISSUE THREAD

**Issue by TheEarthrise** [Rosalie241/texturepack_utils#1]
No matter what hts texture cache i try to extract, it is not working. It aborts directly after the start, here are three attempts with three different texture packs:

```
-> [1/2410] writing SUPER MARIO 64#FEB13915#0#0#00000000_ciByRGBA.png
-> info.width = 0
-> info.height = 0
-> info.format = 0
-> info.texture_format = 0
-> info.pixel_type = 0
-> info.is_hires_tex = 0
-> info.n64_format_size = 0
libpng warning: Image width is zero in IHDR
libpng warning: Image height is zero in IHDR
libpng error: Invalid IHDR data
write_info_to_png failed!
```

```
-> [1/4341] writing Banjo-Kazooie#FF8B90F0#0#0#C6DD78F8_ciByRGBA.png
-> info.width = 0
-> info.height = 0
-> info.format = 0
-> info.texture_format = 0
-> info.pixel_type = 0
-> info.is_hires_tex = 0
-> info.n64_format_size = 0
libpng warning: Image width is zero in IHDR
libpng warning: Image height is zero in IHDR
libpng error: Invalid IHDR data
write_info_to_png failed!
```

```
-> [1/13931] writing PAPER MARIO#13557C62#0#0#08C753DB_ciByRGBA.png
-> info.width = 0
-> info.height = 0
-> info.format = 0
-> info.texture_format = 0
-> info.pixel_type = 0
-> info.is_hires_tex = 0
-> info.n64_format_size = 0
libpng warning: Image width is zero in IHDR
libpng warning: Image height is zero in IHDR
libpng error: Invalid IHDR data
write_info_to_png failed!
```

**Comment by Rosalie241** [comment]
I'm sorry it took so long for me to finally investigate this but I've managed to fix it with this commit: https://github.com/Rosalie241/texturepack_utils/commit/a648f22b1b8b19b6e9b0efa483eb03011f0e709f

I'll try to make a new release with this soon!

**Comment by Rosalie241** [comment]
Fixed in the 1.5 release!

**Comment by TheEarthrise** [comment]
> Fixed in the 1.5 release!

You're awesome, thank you!
